plugins/woocommerce-admin/client/layout/store-alerts/index.js

```diff
@@ -264,6 +266,15 @@ export class StoreAlerts extends Component {
 							</Button>
 						</div>
 					) }
+					<Button
+						onClick={ async () => {
+							await removeNote( alert.id );
+							this.previousAlert();
+							invalidateResolutionForStoreSelector( 'getNotes' );
+						} }
+					>
+						<Icon icon={ close } />
```

Comment:
It would be good to have a tooltip or some other accessible way to know what this button is.

Reply:
How about `Dismiss message`?